app/campaigns/campaigns.service.js

```diff
@@ -0,0 +1,30 @@
+const CampaignDAO = require('./campaigns.db')
+
+const CampaignService = {
+  getCampaigns,
+  getActiveCampaigns,
+  getCompleteCampaigns
+}
+
+async function getCampaigns () {
+  const campaigns = await CampaignDAO.getCampaigns()
+  return {
+    campaigns: campaigns
+  }
+}
+
+async function getActiveCampaigns () {
+  const campaigns = await CampaignDAO.getActiveCampaigns()
+  return {
+    campaigns: campaigns
```

Comment:
Anyway, why are you returning the service response in this format? To me it'd be more natural to return an array. If it's for including it that way in the request response, then that should happen in the controller